azure-pipelines.yml

```diff
@@ -44,6 +44,13 @@ jobs:
     jobName: 'LinuxJava17'
     gradleTasks: 'check -x dependencyCheckAggregate'
 
+- template: build-templates/gradle-common.yml@templates
```

Comment:
@jhouserizer @akomakom should Java 21 replace Java 17 and/or Java 11 here? Or are we good to ahead with 8 PR test jobs?!